lib/interactive.py

```diff
@@ -571,26 +551,7 @@ def copy_packet(self, packet):
             if getattr(data, "SerializeToString", None):
                 data = data.SerializeToString()
 
-            meshPacket = mesh_pb2.MeshPacket()
-            meshPacket.decoded.payload = data
-            meshPacket.decoded.portnum = packet["decoded"]["portnum"]
```

Comment:
`parse_from_packet()` always sets the portnum to `SIMULATOR_APP`, but for `copy_packet()` it should really use the original portnum.

Reply:
Oh no, I'm sorry ^^ Good catch